content/millennium/r4/medications/immunization.md

```diff
@@ -81,6 +81,19 @@ Notes:
 <%= headers status: 200 %>
 <%= json(:R4_IMMUNIZATION_BUNDLE) %>
 
+Note: The examples provided here are non-normative and replaying them in the public sandbox is not guaranteed to yield the results shown on the site.
+
+#### Patient Authorization Request
+
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Immunization?patient=12724066
```

Comment:
Update url to https://fhir-ehr.cerner.com

Reply:
Addressed in commit ID df37656